File: Libraries/Data.Repositories.MongoDB/Models/Location.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

using MongoDB;
using MongoDB.Bson;
using MongoDB.Driver;

namespace Data.Repositories.Models
{
    public class Location
    {
        public Location(ObjectId referenceId, ObjectId tncId, String uiScreen, Double latitude, Double longitude)
        {
            DateTime = DateTime.UtcNow;
            UIScreen = uiScreen;
            UserId = referenceId;
            TNCId = tncId;
            Longitude = longitude;
            Latitude = latitude;
        }

        public DateTime DateTime { get; set; }
        public String UIScreen { get; set; }
        public ObjectId UserId { get; set; }
        public ObjectId TNCId { get; set; }
        public Double Latitude { get; set; }
        public Double Longitude { get; set; }
    }
}

File: Apps/DriveSwitch/app/src/main/java/com/driveswitch/Location.java
package com.driveswitch;

import java.util.Date;

/**
 * Created by Joes-Acer on 1/19/2017.
 */

public class Location
{
    public Location(String userId, String tncId, String uiScreen, Double latitude, Double longitude)
    {
        DateTime = new Date();
        UIScreen = uiScreen;
        UserId = userId;
        TNCId = tncId;
        Latitude = latitude;
        Longitude = longitude;
    }

    public Date DateTime;
    public String TNCId;
    public String UIScreen;
    public String UserId;
    public Double Latitude;
    public Double Longitude;
}
